pygmt/tests/test_coast.py

```diff
@@ -1,94 +1,21 @@
 """
-Tests for coast.
+Tests for fig.coast.
 """
 import pytest
 from pygmt import Figure
 from pygmt.exceptions import GMTInvalidInput
-from pygmt.helpers.testing import check_figures_equal
 
 
 @pytest.mark.mpl_image_compare
-def test_coast():
+def test_coast_iceland():
```

Comment:
I think we should rewrite this test. Below is the baseline image for this test. It's difficult to verify its correctness. Instead, if we plot a regional map like Japan, Hawaii, or Australia, it's much easier to check.

![image](https://user-images.githubusercontent.com/3974108/112732499-ae36ae80-8f10-11eb-9ef9-ddbbf73674c6.png)

Reply:
In 150c45e, I changed the region (had to pick Japan; I miss Okinawa!) and think I made it more readable.